quadhoral-openmeteo/quadhoral_lambda.py

```diff
@@ -0,0 +1,30 @@
+'''Orchestrates weekly ETL pipeline'''
+# pylint: disable=W0612,W0613,E0606
+import logging
+from dotenv import load_dotenv
+from quadhoral_extract import extract
+from quadhoral_transform import transform
+from quadhoral_load import load
+
+def lambda_handler(event, context):
+    '''Orchestrates ETL pipeline for weekly OpenMeteo data'''
+    logger = logging.getLogger(__name__)
+    logging.basicConfig(level=logging.INFO)
+
+    load_dotenv()
+    logging.info('Environment loaded.')
```

Comment:
Solid logging for the lambda handler, and will allow the logging throughout the entire pipeline to run